addons/udes_stock/models/stock_move.py

```diff
@@ -361,6 +361,24 @@ def group_by_key(self):
 
         return self.groupby(lambda ml: ml.u_grouping_key)
 
+    def _prepare_extra_values_for_group(self, pickings, moves):
+
```

Comment:
One empty line before and one empty line after the signature just scatter my eyes.

Reply:
I agree, this is because the docstring is missing.